package.json

```diff
@@ -26,14 +26,22 @@
     "vite": "^2.5.4"
   },
   "dependencies": {
+    "@sendgrid/mail": "^7.4.6",
     "bcryptjs": "^2.4.3",
+    "body-parser": "^1.19.0",
     "dotenv": "^10.0.0",
     "express": "^4.17.1",
+    "express-winston": "^4.2.0",
+    "http-errors": "^1.8.0",
     "joi": "^17.4.2",
+    "jsonwebtoken": "^8.5.1",
```

Comment:
remove extra packages. 

Also, move packages to respective devDeps and deps to optimize things.